utensor_cgen/frontend/tflite.py

```diff
@@ -0,0 +1,398 @@
+      tensor = subgraph.Tensors(idx)
+      buffer_index = tensor.Buffer()
+
+      # buffer_index == 0 if intermediate
+      if buffer_index == 0:
+        continue
+
+      node_name = re.sub(r':\d+', '', self.tensor_names_map[idx].name) + "_Const"
+      dtype = self.tensor_names_map[idx].dtype
+
+      buffer_array = fb_model.Buffers(buffer_index).DataAsNumpy()
+      if isinstance(buffer_array, int):
+        continue  # somehow, sometimes, the buffer contains no data, likely to be an intermediate tensor
+      buffer_content = fb_model.Buffers(buffer_index).DataAsNumpy().view(dtype).reshape(
+        self.tensor_names_map[idx].shape
+      )
+
+      OperationInfo(
+        name=node_name,
+        input_tensors=[],
+        output_tensors=[self.tensor_names_map[idx]],
+        op_type="Const",
+        lib_name="tflm",
+        ugraph=ugraph,
+        op_attr={
+          "value": AttrValueConverter.GenericType(
+            value_name="tensor",
+            value=GenericTensorConverterMixin.GenericType(
+              np_array=buffer_content
+            ),
+          )
+        },
+      )
+
+      self._set_tensor_node(idx, node_name)
+
+  def _build_input_ops(self, fb_model, ugraph):
+    """Find placeholders
+    Attach placeholders to input tensors
+    Note this method will update inputs **inplace**
+    """
+    subgraph = self._get_tflm_get_subgraph(fb_model)
+    subgraph_inputs_indexi = subgraph.InputsAsNumpy()
+    for index in subgraph_inputs_indexi:
+      node_name = self.tensor_names_map[index].name + "_Placeholder"
+      self._set_tensor_node(index, node_name)
+      OperationInfo(
+        name=node_name,
+        input_tensors=[],
+        output_tensors=[self.tensor_names_map[index]],
+        op_type="Placeholder",
+        ugraph=ugraph,
+        lib_name="tflm",
+        op_attr={},
+      )
+
+  def _build_intermediate_ops(self, fb_model, ugraph):
+    """Build all intermediate nodes
+    """
+    subgraphs_len = fb_model.SubgraphsLength()
+    assert subgraphs_len == 1, "only 1 subgraph is supported"
+    subgraph = fb_model.Subgraphs(0)
+    for i in range(0, subgraph.OperatorsLength()):
+      # topological order, op-index defined by schema
+      # BuiltinOperator: https://github.com/tensorflow/tensorflow/blob/031804922d8f4d18b61e3ad077f9f1b69273ff21/tensorflow/lite/schema/schema_v3.fbs#L71
+      op = subgraph.Operators(i)
+      local_op_code = op.OpcodeIndex()
+      global_op_code = fb_model.OperatorCodes(local_op_code)
+      builtin_op_code = global_op_code.BuiltinCode()
+      op_type = self._BUILTIN_OPS[builtin_op_code]
+
+      node_name = str(i) + "_" + op_type
+
+      input_tensor_names = [
+        self.tensor_names_map[input_index] for input_index in op.InputsAsNumpy()
+      ]
+      output_tensor_names = [
+        self.tensor_names_map[output_index]
+        for output_index in op.OutputsAsNumpy()
+      ]
+
+      op_attr = _OP_DATA_FUNC_MAP[op_type](op)
```

Comment:
Please throw error if op_type not in this table, else do best effort default op maybe? We might be able to `import_lib` and do a best effort reflection scan to fill out the `option_dict`. I feel like the frontend should support all ops as best effort, and the actual `is_op_supported()` call should be triggered from the backend